baybe/searchspace/core.py

```diff
@@ -304,22 +304,23 @@ def estimate_product_space_size(parameters: Iterable[Parameter]) -> MemorySize:
     def transform(
         self,
         data: pd.DataFrame,
+        allow_missing: bool = False,
     ) -> pd.DataFrame:
-        """Transform data from experimental to computational representation.
-
-        This function can e.g. be used to transform data obtained from measurements.
-        Continuous parameters are not transformed but included.
+        """Transform parameters from experimental to computational representation.
 
         Args:
-            data: The data to be transformed. Must contain all specified parameters, can
-                contain more columns.
+            data: The data to be transformed. May contain arbitrary additional columns
+                besides columns related to the parameters of the subspace.
```

Comment:
yes, thanks! moved it from the subspace docstring and forgot adjusting. will turn it into just `space` since the subspace methods refer to the docstring and can then just reuse it 👍🏼

Reply:
done